ironfish/src/primitives/block.ts

```diff
@@ -78,6 +78,24 @@ export class Block {
     Assert.isNotUndefined(tx, 'Block has no miners fee')
     return tx
   }
+
+  toCompactBlock(): SerializedCompactBlock {
+    const header = new BlockHeaderSerde(this.header.strategy).serialize(this.header)
```

Comment:
I think we were getting rid of the the serialization strategies. Might have to change this to just `BlockHeaderSerde.serialize`

Reply:
Right on, fixed this up after rebasing